pkg/runtime/scheduler/clients/schedulerclientnotinit.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Rename file to `noinit` as prefix is superfluous through package import name.

Reply:
Sure.